samples/openvx/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 2.8.9)
+
+set(OPENCV_OPENVX_SAMPLE_REQUIRED_DEPS opencv_core opencv_imgproc opencv_imgcodecs opencv_videoio opencv_highgui)
+
+ocv_check_dependencies(${OPENCV_OPENVX_SAMPLE_REQUIRED_DEPS})
+
+if(BUILD_EXAMPLES AND OCV_DEPENDENCIES_FOUND)
+  set(project "openvx")
+  set(name "interop")
+
+  project("${project}_sample")
+
+  ocv_include_modules_recurse(${OPENCV_OPENVX_SAMPLE_REQUIRED_DEPS})
+
+  # building with prebuilt OpenVX package
+  include_directories(${OPENVX_INCLUDE_DIR})
+  link_directories(${OPENVX_ROOT}/lib)
+
+  file(GLOB srcs *.cpp)
+  set(the_target "example_${project}_${name}")
+  add_executable (${the_target} ${srcs})
+
+  ocv_target_link_libraries(${the_target} ${OPENCV_LINKER_LIBS} ${OPENCV_OPENVX_SAMPLE_REQUIRED_DEPS} ${OPENVX_LIBRARIES})
+
+  if(ENABLE_SOLUTION_FOLDERS)
```

Comment:
I'll left that `if` since it's used everywhere in OpenCV when setting `FOLDER` property.